src/main/resources/templates/db_scripts/schema.json

```diff
@@ -374,6 +374,12 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": true
+        },
+        {
+          "fieldName": "deleted_date",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": false
         }
```

Comment:
As I understand it, this section of schema.json creates indexes on JSONB properties, it does not create them on columns. 

I believe `deleted_date` is a database column, and so I don't think this will work as expected.

@julianladisch @adamdickmeiss  @skoczko maybe you could confirm my understanding?

Reply:
Yes, you are right, this creates a trigger for a json field.